brands/ghostery/branding/pref/firefox-branding.js

```diff
@@ -17,157 +19,116 @@ pref("app.update.url.manual", "https://www.ghostery.com/download/");
+****************************************************************************/
 
-// disable tracking protection blocking
+/** TRACKING ***/
+// Disable Firefox's native tracking protection blocking
 pref("browser.contentblocking.category", "custom");
 pref("privacy.trackingprotection.enabled", false);
 pref("privacy.trackingprotection.pbmode.enabled", false);
 pref("privacy.trackingprotection.cryptomining.enabled", false);
 pref("privacy.trackingprotection.fingerprinting.enabled", false);
 pref("privacy.trackingprotection.socialtracking.enabled", false);
 pref("privacy.socialtracking.block_cookies.enabled", false);
+pref("browser.contentblocking.database.enabled", false);
+pref("browser.contentblocking.allowlist.storage.enabled", false);
+pref("browser.contentblocking.cryptomining.preferences.ui.enabled", false);
+pref("browser.contentblocking.fingerprinting.preferences.ui.enabled", false);
 
-/* 0320: disable about:addons' Recommendations pane (uses Google Analytics) ***/
-pref("extensions.getAddons.showPane", false); // [HIDDEN PREF]
-// Recommendations in about:addons' addons pane.
-pref("extensions.htmlaboutaddons.recommendations.enabled", false);
+// Remove addons.mozilla.org from set of domains that extensions cannot access
+pref("extensions.webextensions.restrictedDomains", "accounts-static.cdn.mozilla.net,accounts.firefox.com,addons.cdn.mozilla.net,api.accounts.firefox.com,content.cdn.mozilla.net,discovery.addons.mozilla.org,install.mozilla.org,oauth.accounts.firefox.com,profile.accounts.firefox.com,support.mozilla.org,sync.services.mozilla.com");
 
-/* 0330: disable telemetry
- * the pref (.unified) affects the behaviour of the pref (.enabled)
- * IF unified=false then .enabled controls the telemetry module
- * IF unified=true then .enabled ONLY controls whether to record extended data
- * so make sure to have both set as false
- * [NOTE] FF58+ 'toolkit.telemetry.enabled' is now LOCKED to reflect prerelease
- * or release builds (true and false respectively), see [2]
- * [1] https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/internals/preferences.html
- * [2] https://medium.com/georg-fritzsche/data-preference-changes-in-firefox-58-2d5df9c428b5 ***/
+/** TELEMETRY, STUDIES, EXPERIMENTS, CRASH REPORTS ***/
+// See https://github.com/yokoffing/Better-Fox/blob/a458e14a94ff67111868178639c188a5d08f7038/SecureFox.js#L510
+// for details.
+// [NOTE] We may need re-enable and/or redirect some of these for Ghostery Dawn.
+pref("datareporting.policy.dataSubmissionEnabled", false);
+pref("datareporting.healthreport.uploadEnabled", false);
+pref("browser.ping-centre.telemetry", false);
+pref("default-browser-agent.enabled", false);
+pref("app.shield.optoutstudies.enabled", false);
+pref("app.normandy.enabled", false);
+pref("app.normandy.api_url", "");
 pref("toolkit.telemetry.unified", false);
-pref("toolkit.telemetry.enabled", false); // see [NOTE] above FF58+
-pref("toolkit.telemetry.server", "data:,");
```

Comment:
please bring it back

Reply:
`pref("toolkit.telemetry.enabled", false);` is already in `firefox-brand.js` [here](https://github.com/ghostery/user-agent-desktop/blob/0e258946aa79eaadbb6fbc0d40899e64c0eb638a/brands/ghostery/branding/pref/firefox-branding.js#L78).

I will add `pref("toolkit.telemetry.server", "data:,");` back as it is missing indeed.